build.py

```diff
@@ -99,6 +99,14 @@
 	       "known build platform so that builds are repeatable."
 )
 
+parser.add_argument(
```

Comment:
Oh, and one other thing just occurred to me. The point of GafferHQ/build is to nail down a single build mechanism we use for all our builds, and we'd both really prefer that all the `scl` stuff was hardcoded into the container itself. So I don't think we really want to add a command line option like this : we want to enforce devtoolset 6.